src/components/article/ArticleFullCard.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import { getArticleById } from "../../utils/api";
+import { getDateString } from "../../utils/date";
+import Loading from "../Loading";
+
+function ArticleFullCard({ article_id, article, setArticle }) {
```

Comment:
This article_id should be coming down from useParams - you have the functionality in your router to do it!